classes/Arlo/AuthAPI/XmlDeserializer.php

```diff
@@ -42,14 +42,12 @@ public function deserialize($data) {
         }
 
         $internalErrors = libxml_use_internal_errors(true);
-        $disableEntities = libxml_disable_entity_loader(true);
         libxml_clear_errors();
 
         $dom = new \DOMDocument();
-        $dom->loadXML($data, $this->loadOptions);
+        $dom->loadXML($data, $this->loadOptions | LIBXML_NOENT);
```

Comment:
When loading XML and it is local (not fetched from remote location) always add `LIBXML_NONET` because it is a security measure.